src/Fields/Fields.jl

```diff
@@ -459,17 +459,14 @@ long.
 
 # Example
 ```
-import ClimaCore
-ClimaCore.Fields.truncate_printing_field_types() = true
+import Preferences
+Preferences.@set_preferences!(Pair("TruncateClimaCoreFieldPrinting" => false))
 ```
 =#
-truncate_printing_field_types() = false
-
-function Base.show(io::IO, ::Type{T}) where {T <: Fields.Field}
-    if truncate_printing_field_types()
+import Preferences
+if Preferences.@load_preference("TruncateClimaCoreFieldPrinting", false)
```

Comment:
Preferences are local to the package, so don't need to be qualified
```suggestion
if Preferences.@load_preference("TruncateFieldPrinting", false)
```

Reply:
Ah, good to know. That's what I originally had but thought it'd be safer to qualify. I'll change it back, thanks!